device/thunder_ripple_sdk/src/client/plugin_manager.rs

```diff
@@ -83,11 +83,11 @@ impl ThunderError {
 impl PluginStatus {
     pub fn to_plugin_state(&self) -> PluginState {
         match self.state.as_str() {
-            "activated" => PluginState::Activated,
-            "resumed" => PluginState::Activated,
-            "suspended" => PluginState::Activated,
+            "activated" | "resumed" | "suspended" => PluginState::Activated,
```

Comment:
Need to check Thunder response for call request when the plugin is suspended state.